lib/chef/resource/windows_task.rb

```diff
@@ -59,6 +59,12 @@ class WindowsTask < Chef::Resource
       property :execution_time_limit, [String, Integer], default: "PT72H" # 72 hours in ISO8601 duration format
       property :minutes_duration, [String, Integer]
       property :minutes_interval, [String, Integer]
+      property :disallow_start_on_battery, [TrueClass, FalseClass], default: false,
```

Comment:
Can we change this property name to "start_on_battery" to match the property name below.

Reply:
@tas50 we are setting up the property as per msdn docs so that we can identify or relate them very easily. I will suggest to @dheerajd-msys the property name should be :disallow_start_if_on_battery and :stop_if_on_battery.